src/cell.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+  
+  var Cell = function(value) {
+    this.value = value || null;
+  };
+
+  Cell.prototype.isEmpty = function() {
+    return this.value === null;
+  };
+
+  Cell.prototype.assignValue = function(number) {
+    this.CheckForInvalidValue(number);
+    this.value = number;
+  };
+
+  Cell.prototype.CheckForInvalidValue = function(number) {
```

Comment:
Do we really need all these checks and balances? If so I would argue that it should be done by another module that handles validation and this just handles taking a value and returning it